lib/sandbox.nix

```diff
@@ -49,6 +50,48 @@
   # lib.escapeShellArgs quotes each argument with single quotes. It is safe, but
   # I want to allow use of environment variables passed as arguments.
   quoteShellArgs = lib.concatMapStringsSep " " (wrap "\"" "\"");
+
+  command = lib.concatStringsSep " " (
```

Comment:
I think it is a good idea to separate this expression out. Thank you for the idea.